build/cqs.tsv

```diff
@@ -229,8 +228,8 @@ org.json4s:json4s-native_2.11	3.5.4	compile
 org.json4s:json4s-scalap_2.11	3.5.4	compile
 org.locationtech.jts.io:jts-io-common	1.16.0	compile
 org.locationtech.jts:jts-core	1.16.0	compile
-org.locationtech.sfcurve:sfcurve-api_2.11	0.2.0	compile
-org.locationtech.sfcurve:sfcurve-zorder_2.11	0.2.0	compile
+org.locationtech.sfcurve:sfcurve-api_2.11	0.2.2	compile
+org.locationtech.sfcurve:sfcurve-zorder_2.11	0.2.2	compile
```

Comment:
Doh!  Sorry I missed this in my PR for the update.

Reply:
no worries, since we don't need cqs for LT projects